.github/ISSUE_TEMPLATE/bug_ report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report 🐞
```

Comment:
is this filename correct? looks to have a space after the `_`

Reply:
Oops...not sure how that happened. Fixing.

Reply:
Fixed!